application-boundary.tf

```diff
@@ -63,25 +63,27 @@ resource "cloudfoundry_service_instance" "k8s_cluster" {
 #   ]
 # }
 
-data "cloudfoundry_org" "gsa" {
-  name = var.broker_space.org
+# For now we are using a hand-provisioned user-provided service, not managed by Terraform
+data "cloudfoundry_space" "broker-space" {
+  name     = var.broker_space.space
+  org_name = var.broker_space.org
```

Comment:
Is this just to make the ssb more module to be used by others?  Would anyone ever use this repo?  I thought this was specific to us?